web/unit_tests/cases/engine_chirality.js

```diff
@@ -1,9 +1,9 @@
 var assert = chai.assert;
 
 describe('Engine - Chirality', function() {
-  before(function(done) {
-    this.timeout(kmwconfig.timeouts.scriptLoad);
+  this.timeout(kmwconfig.timeouts.scriptLoad);
 
+  before(function(done) {
```

Comment:
The placement of this timeout's been causing some CI instability.  This should help with that, hopefully.